packages/application-shell/src/components/login-locked/login-locked.spec.js

```diff
@@ -1,46 +1,10 @@
 import React from 'react';
-import { shallow } from 'enzyme';
-import LockedAccount, { HelpDeskLink, ResetPasswordLink } from './login-locked';
+import { render, waitForElement } from '../../test-utils';
+import LockedAccount from './async';
 
-describe('LockedAccount', () => {
-  describe('rendering', () => {
-    let wrapper;
-    describe('rendering', () => {
-      beforeEach(() => {
-        wrapper = shallow(<LockedAccount />);
-      });
-      it('should match snapshot', () => {
-        expect(wrapper).toMatchSnapshot();
-      });
-      it('should render layout component', () => {
-        expect(wrapper).toRender('ServicePageResponseLayout');
-      });
-    });
-  });
-});
-describe('HelpDeskLink', () => {
-  let wrapper;
-  describe('rendering', () => {
-    beforeEach(() => {
-      wrapper = shallow(<HelpDeskLink />);
-    });
-    it('should match snapshot', () => {
-      expect(wrapper).toMatchSnapshot();
-    });
-  });
-});
-describe('ResetPasswordLink', () => {
-  describe('rendering', () => {
-    let wrapper;
-    describe('rendering', () => {
-      beforeEach(() => {
-        wrapper = shallow(
-          <ResetPasswordLink adminCenterUrl="https://ac.ct.com" />
-        );
-      });
-      it('should match snapshot', () => {
-        expect(wrapper).toMatchSnapshot();
-      });
-    });
-  });
+it('renders maintenance page for locked account', async () => {
+  const { getByText, queryByText } = render(<LockedAccount />);
+  await waitForElement(() => getByText(/Account Locked/i));
+  expect(queryByText(/reset your password/i)).toBeDefined();
```

Comment:
nitpick: just wondering why you are using queryByText here? It's mostly only used to test for things not to be there?

how about

   expect(getByText(/reset your password/i)).toBeInTheDocument();

Reply:
As far as I understand, `getByText` throws an error if it does not find the element, therefore it's usually used within a e.g. `waitForElement` call.

The `queryByText` does not throw, so we can use a normal `expect`.

Correct me if I'm wrong though... 🙃

Reply:
ah I'm not really familiar with waitForElement. good call 👍